### PR TITLE
update for debug info in documentation

### PR DESCRIPTION
**Pull request by kahmed10**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +68,13 @@ And can be installed by using the 'install' target:
 
 ` cmake --build . --config Release --target install ` **OR** ` make install `
 
-This will install the library to the `CMAKE_INSTALL_PREFIX` path that was set. 
+This will install the library to the `CMAKE_INSTALL_PREFIX` path that was set.
+
+Alternatively, to build the library using the 'Debug' configuration:
+
+` cmake --build . --config Debug `
```
Comment (pfultz2) [inline README.md:75]: That will only build debug for Visual Studio projects. `CMAKE_BUILD_TYPE` needs to be set to `Debug`.

Comment (kahmed10) [reply README.md]: I removed that line and referred to the `CMAKE_BUILD_TYPE` variable instead. If there's a better way of explaining it, feel free to add a suggestion.

File: README.md [hunk 2]
```diff
@@ -68,7 +68,11 @@ And can be installed by using the 'install' target:
 
 ` cmake --build . --config Release --target install ` **OR** ` make install `
 
-This will install the library to the `CMAKE_INSTALL_PREFIX` path that was set. 
+This will install the library to the `CMAKE_INSTALL_PREFIX` path that was set.
+
+To build the library using the 'Debug' configuration, the cmake environment variable `CMAKE_BUILD_TYPE` can be modified to `Debug`:
```
Comment (pfultz2) [inline README.md:73]: I would say 'To build a debug version of the library, the cmake variable `CMAKE_BUILD_TYPE` can be set to `Debug`.

File: README.md [hunk 3]
```diff
@@ -68,7 +68,11 @@ And can be installed by using the 'install' target:
...
+
+` cmake -DCMAKE_BUILD_TYPE=Debug . ; make `
```
Comment (pfultz2) [inline README.md:75]: Just write the cmake line without the make command.

File: README.md (hunk 2 above)
Comment (kahmed10) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (kahmed10) [reply README.md]: fixed

**Pull request merged**